### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by nuzeko** [pizzaboxer/bloxstrap#1050]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

I literally cant download it cause of that error (System.Exception: Signature does not match!) it happens while im installing it
![image_2023-12-21_174140616](https://github.com/pizzaboxer/bloxstrap/assets/154538073/37c15ff3-88f7-4eac-b784-827535955fa8)

**Comment by pizzaboxer** [comment]
As per the preliminary instructions, have you read https://github.com/pizzaboxer/bloxstrap/wiki/%22Signature-does-not-match!%22-error-when-launching-Roblox?

**Comment by nuzeko** [comment]
> As per the preliminary instructions, have you read https://github.com/pizzaboxer/bloxstrap/wiki/%22Signature-does-not-match!%22-error-when-launching-Roblox?

Yes i have and i've tried downloading it a month ago aswell

**Comment by BokBoy** [comment]
Do you have another computer that you can try?

Or you could try a more stable wifi like using Ethernet?

**Comment by pizzaboxer** [comment]
> Yes i have and i've tried downloading it a month ago aswell

It's always been doing this for the past month? Does the standard Roblox launcher even work?

**Comment by nuzeko** [comment]
> > Yes i have and i've tried downloading it a month ago aswell
> 
> It's always been doing this for the past month? Does the standard Roblox launcher even work?

Yes it does

**Comment by pizzaboxer** [comment]
Open a Command Prompt window, and run the following command:

`curl https://setup.rbxcdn.com/version-48a28da848b7420d-RobloxApp.zip -o %userprofile%\Downloads\RobloxApp.zip`

What happens when you run it? Show a picture of the console window.

**Comment by nuzeko** [comment]
> Open a Command Prompt window, and run the following command:
> 
> `curl https://setup.rbxcdn.com/version-48a28da848b7420d-RobloxApp.zip -o %userprofile%\Downloads\RobloxApp.zip`
> 
> What happens when you run it? Show a picture of the console window.

![image_2023-12-22_130631332](https://github.com/pizzaboxer/bloxstrap/assets/154538073/7e5d5118-f01b-4722-a7c2-49798c300eb8)
this is what happens

**Comment by pizzaboxer** [comment]
Yeah, that's what I thought. I'm hard-pressed to believe that Roblox somehow works.

If you go to https://setup.rbxcdn.com/ and [view the certificate details for it](https://www.howtogeek.com/292076/how-do-you-view-ssl-certificate-details-in-google-chrome/), what does it show?

**Comment by nuzeko** [comment]
> Yeah, that's what I thought. I'm hard-pressed to believe that Roblox somehow works.
> 
> If you go to https://setup.rbxcdn.com/ and [view the certificate details for it](https://www.howtogeek.com/292076/how-do-you-view-ssl-certificate-details-in-google-chrome/), what does it show?

![image](https://github.com/pizzaboxer/bloxstrap/assets/154538073/d060f747-3e81-454c-b811-f06e7a69cf62)
 this is what it shows

**Comment by pizzaboxer** [comment]
I asked you to view the certificate details. Follow the steps in the article and post a screenshot of the window that's shown.

**Comment by nuzeko** [comment]
> I asked you to view the certificate details. Follow the steps in the article and post a screenshot of the window that's shown.

Sorry for the delay, I wasn't home. This is what it shows.
![image_2023-12-26_145313579](https://github.com/pizzaboxer/bloxstrap/assets/154538073/5d027d26-5155-4ede-a1d4-f94e46062f55)

**Comment by pizzaboxer** [comment]
Okay, that looks fine. What? What is going on here? I'm not entirely sure what I can do about this,

**Comment by pizzaboxer** [comment]
I was meant to close this a long while ago since there isn't actually anything that I can do about this as of right now - we're changing some functionality that could possibly lead to this being fixed, but that's about it.
